Jim-Hodapp-Coaching/refactor-platform-fe
# Mobile breakpoint css updates and related miscellaneous

## Description
This PR makes mobile breakpoint css updates and related miscellaneous

**PR #47 must land first**

#### GitHub Issue: Resolves #48 

### Changes
* Add Refactor RGB logo as the favicon for the site
* Add a large max width to the page and all components on the coaching session page (1536px)
* Always show the DashboardContainer component for all media breakpoints
* Update mobile-nav and command-menu items to non-template links
* Make sure the login form always shows at all breakpoints and tweak the top padding for the small breakpoint

### Screenshots / Videos Showing UI Changes (if applicable)
* Minor multiple tweaks, no screenshot supplied

### Testing Strategy
1. Notice that the favicon in your browser tab is now an RGB Refactor logo
2. Resize the login and dashboard pages and notice the main content remains and is a responsive layout design
3. Resize the dashboard or coaching session page and notice the mobile menu appear
4. The menu should only have three choices, Dashboard, Profile and Log out

### Concerns
* Log out from the mobile-nav doesn't work yet. We'll need a more universal logout method to call that isn't just a part of user-nav alone.
